packages/rest/src/routes/swagger.json

```diff
@@ -1,111 +1,110 @@
 {
```

Comment:
Would be possible to have all these generated files inside a `generated` folder? Just to keep it clear what is generated by computers and humans :).

Reply:
Sure 👍